Training5/src/task1/src/actions/todo.js

```diff
@@ -0,0 +1,65 @@
+export const GetTodos = (todos)=>{
+    return {
+        type: 'GET_TODOS',
+        payload: todos,
+    }
+}
+
+export const AddNewTodo = (todo)=>{
+    return {
+        type: 'ADD_NEW_TODO',
+        payload: todo,
+    }
+}
+
+export const SetCompletedTodo = (todo)=>{
+    return {
+        type: 'SET_COMPLETED_TODO',
+        payload: todo,
+    }
+}
+
+export const DeleteTodo = (todo)=>{
+    return {
+        type: 'DELETE_TODO',
+        payload: todo,
+    }
+}
+
+export const EditTodo = (todo)=>{
+    return {
+        type: 'EDIT_TODO',
+        payload: todo,
+    }
+}
+
+export const OpenFormEdit = (todo)=>{
```

Comment:
@phuongluongZigvy you can create ONE action creator for and `isOpen`: boolean, `editingTodoId`: string in the payload, we must reuse